packages/authorize/useAuthorize.js

```diff
@@ -7,6 +7,7 @@ export default (
 ) => {
   const [authorized, setAuthorized] = useState(false);
   const [loading, setLoading] = useState(true);
+  const [currentRegion, setCurrentRegion] = useState('');
```

Comment:
let's add a test that you can pull back the current region from the `useAuthorize` hook: https://github.com/Availity/availity-react/blob/master/packages/authorize/tests/useAuthorize.test.js#L55